config/software/datadog-agent-integrations.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/ostools.rb'
+
+name 'datadog-agent-integrations'
+
```

Comment:
`dependency 'datadog-agent'`, before/after `dependency 'pip'` should do the trick.